app/assets/stylesheets/lifehack_style.scss

```diff
@@ -0,0 +1,116 @@
+.form-text {
+  float: left;
+  font-size: 15px;
+  font-weight: bold;
+  margin: 15px;
+}
+
+.lifehack-details {
+  max-width: 300rem;
+  padding: rem-calc(50);
+}
+
+.panel {
+
+  h5 {
+    font-size: 30px;
+    text-align: center;
+  }
+
+  h2 {
+    border-style: double;
+    color: $primary-color;
+    font-size: 60px;
+    text-align: center;
+    }
+
+  table tr td {
+    background-color: $smoke;
+    padding: 13px 0.875rem 13px 2.875rem;
+  }
+
+  .creator-name {
+    padding: 0;
+  }
+
+  .creator-image {
+    padding: 5;
+  }
+
+  .lifehack-description {
+    font-size: 20px;
+  }
+}
+
+.description {
+  p {
+    color: $primary-color;
+    font-size: 50px;
+    padding: 25px 15px 100px 20px;
+    text-align: justify;
+    width: 300px;
+  }
+
+  table th {
+    background-color: $smoke;
+    font-size: 18px;
+    margin-left: auto;
+    margin-right: auto;
+    padding: 3px 5.875rem 21px 4.875rem;
+    text-align: center;
+  }
+
+  table td {
```

Comment:
it doesn't seem like we need the table selector here (but keep th / td), maybe if we take that out then these hound violations will go away